lib/preprocess.js

```diff
@@ -59,6 +59,10 @@ export const preprocess = async ({
     { total: measurements.length, valid: validMeasurements.length }
   )
 
+  const ok = validMeasurements.reduce((c, m) => m.retrievalResult === 'OK' ? c + 1 : c, 0)
```

Comment:
`ok` is a vague variable name, what about `okMeasurementsCount`?

Reply:
https://github.com/filecoin-station/spark-evaluate/pull/82